### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Screen Reader] [MAS1.3.1] [Navigation landmark is missing and multiple section region landmark is defined for the Navigation page.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#710]
**User Experience:**
User with vision disability will not be able to identify the navigation region as the navigation landmark is not defined. Also there are multiple section region landmark with differentiating name.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0

**Attachments:**
[MAS1.3.1 Landmarks.zip](https://github.com/OfficeDev/script-lab/files/1456923/MAS1.3.1.Landmarks.zip)

**Repro Steps:**
1.Open the URL: https://script-lab.azureedge.net/#/edit/excel
2. Navigate to the page in the landmark mode.
3. Verify that there is no navigation landmark defined and also there are multiple landmarks present with no correct name defined in it.

**Current Behavior:**
Navigation landmark is not defined. And there are multiple section region landmark defined on the page.

**Expected Behavior:**
Navigation landmark should be defined so that user can easily navigate to those controls. Also all the landmarks should have meaningful name through which it can be differentiated.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.3.1
